challenge_4/python/CindyB/src/Challenge4.py

```diff
@@ -0,0 +1,15 @@
+import sys
+
+from Tree import Tree
+from TreeNode import TreeNode
+
+input = [4,2,7,1,3,6,9]
+myTree = Tree(TreeNode(input[0]))
+input.remove(input[0])
+
+for value in input:
+    myTree.insert(value,myTree.GetRoot())
```

Comment:
It's odd that inserting to a tree object would require passing in that same tree's root node. This could instead be abstracted away from the caller.

in class Tree:
```
def insert(self, value):
  self._insert(value, self.GetRoot())

def _insert(self, value, node):
  # Same as previous insert, except recursively calling _insert.
```